samples/Network/no-public-ip-except-for-one-subnet/azurepolicy.json

```diff
@@ -5,7 +5,8 @@
             "subnetIds": {
                 "type": "Array",
                 "metadata": {
-                    "description": "List of Subnets",
+                    "description": "The subnetIds parameter must be provided with a list of subnets in format of resource ID (e.g /subscriptions/\{subscription_id\}/resourcegroups/\{resource_group\}/providers/microsoft.network/virtualnetworks/\{vnet\}/subnets/\{subnet-name\}
```

Comment:
The '{' characters don't need to be escaped inside a string.  Also, all iterations of this string are missing the closing ")" from the opening "(e.g.".  Once these are done it will be good to go.

Reply:
This has now been fixed.